app/index.js

```diff
@@ -34,6 +33,7 @@ app.use(sass({
   src: 'www/styles'
 }));
 
+
```

Comment:
Be a little more conscious about manipulating spaces. You can see the total changes in the "Files changed" tab above.